### PR TITLE
 Krypto - Metrics webapp 

### PR DESCRIPTION
**Pull request by Moses-chibuike**
- Built the project as a single-page application (SPA) with React and Redux.
- Stored the data retrieved from the API in the Redux store.
- Deployed the web app on render and added the live link to the READme file
- Filtered the data that was retrieved from the API using a Filter stateless component.
- Made every page (the main page and the pages for each item) have a unique route within the SPA.
- Recorded a 4mins+ video describing the project and added the video link to the READme file

### CODE REVIEW

File: src/Pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <input
+          type="text"
+          placeholder="Search An Exchange"
+          onChange={(e) => handleSearch(e.target.value)}
+        />
+      </div>
+      <div className="exchange-container">
+        {exchangeList.map((exchange) => (
+          <ExchangeDetails
+            key={exchange.id}
```
Comment (thecodechaser) [inline src/Pages/Home.js]: - [ ] Please make sure every page (the main page and the pages for each item) should have a unique route as requested in the project requirements. Currently, All crypto's pages are showing on the same route`(/details`, For a unique route you can use the names of cryptos and pass them in the URL to use in the route. You can read more about React dynamic routing [here](https://dev.to/shubhamtiwari909/dynamic-routes-in-react-491g)

image

image

image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,169 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Video Demo <a name="demo"></a>
+
+watch demo of this amazing project
+
+Feel free to check the [watch video](https://drive.google.com/file/d/1rg7Y77ZtTDU8JAAyZPCSCMvNDL7Revey/view?usp=share_link).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (thecodechaser) [inline README.md]: - [ ] Please make sure in the presentation video also to show some piece of code that you're proud of as requested in the project requirements. (You need to record the presentation video again).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,169 @@
...
+<div align="center">
+
+  <h3><b>Krypto</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
```
Comment (thecodechaser) [inline README.md:12]: - [ ] Please make sure the project is deployed online and add the live demo link to the README file as requested in the project requirements.

image

File: README.md [hunk 4]
```diff
@@ -1,2 +1,169 @@
...
+
+If you like this project please give it a thumb up.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse Inc. for providing template and other guidelines for this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (thecodechaser) [inline README.md:169]: - [ ] Please make sure to give appropriate credit to the author in the README for the original design idea as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Moses-chibuike👋, 

### You are doing a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight 🎉
- All tests are passed. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Moses-chibuike 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes and well done on this great project overall 💯 

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**